projects/ngrx.io/content/guide/component/let.md

```diff
@@ -1,82 +1,98 @@
+In case of `0` (falsy value), the component would be hidden.
 
-The `*ngrxLet` directive takes over several things while making it more convenient and safe to work with streams in the template.
+The `*ngrxLet` directive takes over several things and makes it more convenient
+and safe to work with streams in the template:
 
 ```html
-<ng-container *ngrxLet="observableNumber$ as n">
-  <app-number [number]="n">
-  </app-number>
+<ng-container *ngrxLet="number$ as n">
+  <app-number [number]="n"></app-number>
 </ng-container>
 
-<ng-container *ngrxLet="observableNumber$; let n">
-  <app-number [number]="n">
-  </app-number>
+<ng-container *ngrxLet="number$; let n">
+  <app-number [number]="n"></app-number>
 </ng-container>
 ```
 
-In addition to that it provides us information from the whole observable context.
-We can track the observable notifications:
+### Tracking Different Observable Events
 
-- next value
-- error value
-- completion state
+In addition to that it provides us information from the whole observable context.
+We can track next, error, and complete events:
 
 ```html
-<ng-container *ngrxLet="observableNumber$; let n; let e = $error, let c = $complete">
-  <app-number [number]="n"  *ngIf="!e && !c">
+<ng-container *ngrxLet="number$ as n; let e = $error; let c = $complete">
+  <app-number [number]="n" *ngIf="!e && !c">
   </app-number>
-  <ng-container *ngIf="e">
-  There is an error: {{e}}
-  </ng-container>
-  <ng-container *ngIf="c">
-  Observable completed: {{c}}
-  </ng-container>
+
+  <p *ngIf="e">There is an error.</p>
+  <p *ngIf="c">Observable is completed.</p>
+</ng-container>
+```
+
+### Using Suspense Template
+
+There is an option to pass the suspense template that will be displayed
+when an observable is in a suspense state:
+
+```html
+<ng-container *ngrxLet="number$ as n; suspenseTpl: loading">
+  <app-number [number]="n"></app-number>
 </ng-container>
+
+<ng-template #loading>
+  <p>Loading...</p>
+</ng-template>
 ```
 
+<div class="alert is-helpful">
+
+Observable is in a suspense state until it emits the first event (next, error, or complete).
+
+</div>
+
+In case the new observable is passed to the `*ngrxLet` directive in runtime,
+the suspense template will be displayed again until the new observable emits the first event.
+
 ## Included Features
 
-- Binding is always present. (`*ngIf="truthy$"`)
-- Takes away the multiple usages of the `async` or `ngrxPush` pipe
-- Provides a unified/structured way of handling `null` and `undefined`
-- Triggers change-detection differently if `zone.js` is present or not (`ChangeDetectorRef.detectChanges` or `ChangeDetectorRef.markForCheck`)
-- Triggers change-detection differently if ViewEngine or Ivy is present (`ChangeDetectorRef.detectChanges` or `ɵdetectChanges`)
-- Distinct same values in a row (distinctUntilChanged operator),
-                                                                 
+- Binding is always present.
```

Comment:
I don't know what this means without the `ngIf` example.
Maybe we should just leave it out of the features, or compare it against `ngIf` ?

Reply:
Maybe: `Binding is present even for falsy values.` makes more sense. What do you think?

Reply:
or: Binding is present even for falsy values. (See ["Comparison with `*ngIf` and `async`"](#comparison-with-ngif-and-async) section)

👀

Reply:
Both options are good to me 👍